src/components/tables/RowActions/ProgressDialog.tsx

```diff
@@ -60,7 +60,7 @@ function ProgressDialog({
                 </Stack>
             </DialogContent>
 
-            <DialogActions>
+            <DialogActions sx={{ p: '16px 24px' }}>
```

Comment:
why are we using pixels here?

Reply:
This style is copied and pasted from a legacy dialog and I will adjust this instance. To remove all others, a dedicated PR should be opened.